src/components/icons/Sort.tsx

```diff
@@ -148,8 +169,12 @@ const IconDesc = ({ fill, size = 18, style = {}, cssRaw, animated, animationComp
 /** Sort Icon Component with Conditional Lottie Animation. */
 const SortIcon = ({ size = 18, style = {}, cssRaw, animated, animationComplete }: IconType) => {
   const direction = useSelector(getCursorSortDirection)
+
+  // Disable animation for HOME_PATH or no direction
+  const isAnimated = direction === 'Desc' ? true : animated && direction !== null
```

Comment:
An OR might be simpler than the ternary here:

```suggestion
  const isAnimated = direction === 'Desc' || (animated && direction !== null)
```